### DISCUSSION TITLE
Which is the best way to send data?

### DISCUSSION THREAD

**Discussion by funnypinky** [danak6jq/Seeed-LoRa-E5#29] [Q&A]
Hello,
i try your project for my application. The end node join to the network, its work fine. But i don't find a example to send data to the network. I search in the example from Seeed Studio Wiki, but i don't find an idea.
Can you get a code snippet to send data?
Very thanks,
Funnypinky

**Comment by danak6jq** [comment]
Please refer to LoRaWAN/App/lora_app.c:SendTxData() - that's exactly what this does.
